stormpath-export

```diff
@@ -40,16 +39,16 @@ class StormpathExport(object):
 
     EXPORTS = ['tenants', 'applications', 'directories', 'groups', 'organizations', 'accounts']
 
-    def __init__(self, base_url=None):
+    def __init__(self):
         """Initialize our Stormpath client, or die tryin' >:)"""
         if exists(CONFIG_FILE):
             credentials = loads(open(CONFIG_FILE, 'rb').read())
             self.client = Client(api_key={
                 'id': credentials.get('stormpath_api_key_id'),
                 'secret': credentials.get('stormpath_api_key_secret'),
-            }, base_url=base_url or None)
+            }, base_url=credentials.get('stormpath_base_url') or None)
```

Comment:
I think this should be `credentials.get(...) or base_url or None` in that order. So we try to get it from the credentials file, or the CLI options